nightingale_dispatcher/src/nightingale_dispatcher/navigate_task.py

```diff
@@ -28,7 +29,7 @@ def execute(self, room_number, sublocation):
         self.action_client.send_goal(goal)
         self.action_client.wait_for_result()
         return (
-            Task.SUCCESS
```

Comment:
I think all tasks should return `Task.SUCCESS` rather than the `TaskWith...` status for compatibility